Frontend/voitheia-App/src/app/components/user/login/login.component.ts

```diff
@@ -15,7 +15,7 @@ export class LoginComponent implements OnInit {
   faPaperPlane = faPaperPlane;
   faUserPlus = faUserPlus;
   
-  constructor(private _authService: AuthService, private _navigationService: NavigationService) {
+  constructor(private _authService: AuthService, public _navigationService: NavigationService) {
```

Comment:
_navigationService should not be public. If needed in the HTML we should present a wrapper function.

Reply:
In HTML we should use `routerLink="/about/idea" routerLinkActive="active"`

Reply:
changed